compiler/modules/front/codeGen.bal

```diff
@@ -575,6 +578,30 @@ function codeGenAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environ
     }
 }
 
+function codeGenCompoundAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environment env, CompoundAssignStmt stmt) returns CodeGenError|StmtEffect {
+    var { lValue, rexpr , op, pos} = stmt;
```

Comment:
Please keep the spaces `{ lValue, rexpr, op, pos }`.
Check the other places too.

Reply:
fixed